src/Orleans.Transactions/State/TransactionalState.cs

```diff
@@ -187,15 +190,18 @@ Task ITransactionalResource.Abort(long transactionId)
         /// </summary>
         async Task ITransactionalResource.Commit(long transactionId)
         {
+            if (this.logger.IsEnabled(LogLevel.Debug)) this.logger.Debug("Commit {0}", transactionId);
+
             // Learning that t is committed implies that all pending transactions before t also committed
             if (transactionId > this.metadata.StableVersion.TransactionId)
             {
                 this.highCommitTransactionId = Math.Max(this.highCommitTransactionId, transactionId);
                 try
                 {
                     bool success = await this.storageExecutor.AddNext(() => GuardState(() => PersistCommit(transactionId)));
-                } catch(Exception)
+                } catch(Exception ex)
                 {
+                    this.logger.Error(OrleansTransactionsErrorCode.Transactions_CommitFailed, $"Commit {transactionId} failed.  Recovering.", ex);
```

Comment:
I guess it's a matter of opinion, but are failed commits errors? I think we can leave this on Error level for now, but failed commits are somewhat expected and normal.

Reply:
Probably should be info level, since system will retry.

Reply:
I think it should be Info level. Failed commit is expected and handled by tx system.